hexrd/unitcell.py

```diff
@@ -1246,6 +1209,50 @@ def color_orientations(self,
 
         self.stiffness = C
 
+    def is_editable(self, lp_name):
+        """
+        @author Saransh Singh, Lawrence Livermore National Lab
+        @date 03/17/2021 SS 1.0 original
+        @details check if a certain field in the lattice parameter 
+        is editable. this depends on the space group number or the
+        lattice class
+        """
+        res = False
+        _lpnamelist = list(_lpname)
```

Comment:
Where is this `_lpname` variable defined?

Reply:
It's defined in `unitcell` module [here](https://github.com/HEXRD/hexrd/blob/669c2334ab39c9bc069e7818e63ff1530f8a32aa/hexrd/unitcell.py#L1592)